app/components/sidebars/main/channels_list/list/index.js

```diff
@@ -2,46 +2,69 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
 
 import {General} from 'mattermost-redux/constants';
 import {
     getSortedUnreadChannelIds,
-    getSortedFavoriteChannelIds,
-    getSortedPublicChannelIds,
-    getSortedPrivateChannelIds,
-    getSortedDirectChannelIds,
+    getOrderedChannelIds,
 } from 'mattermost-redux/selectors/entities/channels';
 import {getCurrentUserId, getCurrentUserRoles} from 'mattermost-redux/selectors/entities/users';
 import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
 import {getTheme, getFavoritesPreferences} from 'mattermost-redux/selectors/entities/preferences';
+import {getSidebarPreferences} from 'mattermost-redux/selectors/entities/sidebar';
```

Comment:
is sidebar an entity? this should have been done in the preferences entity instead.

I've created a ticket here to fix it https://mattermost.atlassian.net/browse/MM-11531